### ISSUE TITLE
Sounds not played in Build Monitor tab

### ISSUE THREAD

**Issue by ciaranmccarthy** [jan-molak/jenkins-build-monitor-plugin#159]
When displaying the Build Monitor Plugin in a tab in a web browser, sounds played using the Jenkins Sounds plugin aren't played in the Build Monitor Plugin tab.

Sounds were played when using the previous version of the Build Monitor Plugin (1.6+build.149).

Version information:
Build Monitor Plugin 1.6+build.150
Jenkins Sounds plugin 0.4.3 (https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+Sounds+plugin)
Jenkins 1.618
Google Chrome 43.0.2357.130 m
Windows 7 Service Pack 1

(We have a dedicated screen in our development team showing current build statuses, and playing a happy 'ding!' sound when everything goes well, or a loud obnoxious 'buzz' when everything doesn't!)

**Comment by jan-molak** [comment]
Thanks for reporting this, I'll have a look.
I wasn't aware that Build Monitor is being used with the Sounds plugin.
Sounds like a cool combo! (no pun intended ;) )

**Comment by ciaranmccarthy** [comment]
Magic, thanks for having a look. Yeah, the screen and the sounds are really useful. Apart from when the boss comes in and asks why nearly everything is red. :D

The team here at beyontics say thanks for your excellent plugin! Especially our boss.

**Comment by jan-molak** [comment]
@ciaranmccarthy - Thanks, your kind words are much appreciated :) 

I'm happy to announce that as of [v1.6+build.154](https://smartcode-opensource.ci.cloudbees.com/job/jenkins-build-monitor-plugin/154/) you and your team at beyontics no longer have to live in silence and may enjoy the "loud obnoxious 'buzz'" again!

Although I do wish you and your boss that you don't have to hear that sound too often  :wink:

**Comment by jordy93** [comment]
I think this feature isn't working anymore? I hear nothing on my build monitor tab :(

**Comment by jan-molak** [comment]
@jordy93 - what version of Build Monitor/Jenkins/Jenkins Sounds Plugin are you using please?

**Comment by jordy93** [comment]
Hi Jan,

Thanks for the quick response, I'm using these verions:

Build Monitor View - 1.11+build.201701152243
Sounds plugin - 0.5
Jenkins - 2.32.1

**Comment by bascht** [comment]
@jan-molak @jordy93 we trapped into the same issue: 

The sound monitor plugin injects a bit of JavaScript that still requires `prototype.js`:

```javascript
new Ajax.Request("/sounds/getSounds", 
{
        parameters: { version: VERSION },
        onSuccess: function(rsp) {
        onSuccess(eval('x='+rsp.responseText))
    },
  onFailure: onFailure
});
```

And it looks like 321cfb2fd5a removed Prototype.js to fix #217. 

So this is a bit of a dead end.
It might be easier to just fix that bit of JS for the Sound Plugin. :wink:

**Comment by jan-molak** [comment]
Lol, cool, thanks for the update! Yes, that sounds like a sensible approach :-)
